src/assets/js/utils/listeners.js

```
@@ -0,0 +1,37 @@
+
+module.exports.screenWidgetListener = (component) => {
+	addManagedListener(component, window, 'resize', debounce(throttle(() => component.set('screenWidth', window.innerWidth), 200)));
+};
+
+module.exports.stickySidebarScrollListener = (component) => {
+	let sidebar = document.querySelector('.page-content_side-menu');
+
+	if (sidebar) {
+		let top = sidebar.getBoundingClientRect().top + document.body.scrollTop;
+
+		let handleStickySidebarScroll = () => {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'handleStickySidebarScroll' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)